rust_dev_preview/iam/README.md

```diff
@@ -1,58 +1,72 @@
-# AWS SDK for Rust code examples for IAM
+# IAM code examples for AWS SDK for Rust
 
-## Purpose
+## Overview
 
-These examples demonstrate how to perform several AWS Identity and Access Management (IAM) operations using the developer preview version of the AWS SDK for Rust.
+Shows how to use the AWS SDK for Rust to manage Amazon IAM resources.
```

Comment:
Amazon IAM --> AWS Identity and Access Management (IAM)

Reply:
fixed